plugins/plugin-k8s/src/test/k8s/status.ts

```diff
@@ -48,7 +44,7 @@ const doHeadless = (ctx: common.ISuite, impl: CLI) => {
   it('should delete the new pod by yaml', () => {
     return impl.do(`kubectl delete -f ${ROOT}/data/k8s/headless/pod.yaml`, ctx.app)
       .then(impl.expectOK('pod "nginx" deleted'))
-      .then(waitTillNone('pods', impl))
+      .then(() => waitTillNone('pods', impl)(ctx.app))
```

Comment:
why doesn't impl.expectOK return an `app`? the UI impl does. is the problem that the headless impl doesn't?

Reply:
headless impl.expectOK() returns the actual output